### ISSUE TITLE
[v3.14.0b]  autosubmit recovery EXPID --all -s fails

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#851]
In GitLab by @vlap on Jul 28, 2022, 12:28

Hi @dbeltrankyl,

I was using autosubmit/3.14.0b-foss-2015a-Python-2.7.9 when I tried to run `autosubmit recovery EXPID --all -s` to recover jobs status from mn4, to continue the experiment after autosubmit stopped for some reason. But the command failed with this error
```
[ERROR] Trace: 'NoneType' object has no attribute 'send_command'
 [CRITICAL] Couldn't restore the job_list or packages, check if the filesystem is having issues [eCode=7040]
```
I loaded the default version (3.13.0) and ran the same command to update the jobs statuses without an issue, it worked as expected.

Not sure if this is a bug or something was changed for a good reason. Just FYI.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 28, 2022, 12:48

Thanks for the report, I'll fix it soon. 

It is a bug, seems like it tries to cancel an active job without connecting to the platform.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 28, 2022, 13:17

Updating 3.14.0b... should be fixed in about 5 min ~

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Oct 5, 2022, 13:24

It is not an entirely related issue, but @molidg was not able to use setstatus with the 3.14.0+ version due to a bug in the automatic cancel.


It is derived from the slurms issues of weekend #879 , seems that some jobs had an id "0" and setstatus was crashing due to that. It has been fixed on 3.14.0b. 

In any case, now is fixed and I'll close the issue.
